homeworks/eduardo-griboldi/contact.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Eduardo Garcia Riboldi</title>
+</head>
+
+<body>
+   
+        <a href="file:///Users/chepegarciariboldi/Desktop/repoFrontEnd/SOLVD-FE-training/homeworks/eduardo-griboldi/home.html"> Home</a>
+        <a href="file:///Users/chepegarciariboldi/Desktop/repoFrontEnd/SOLVD-FE-training/homeworks/eduardo-griboldi/contact.html"> Contact</a>
+    <br/>
+    
+    <h1>Contact Me</h1>
+    <form>
+        <label>Full name: 
+            <input id="First-Name" type="text"  placeholder="Your name..."/> 
+            <input id="Last-Name" type="text" placeholder="Your last name..."/> 
```

Comment:
in general its better to use camelCase naming convention for values in attributtes
https://www.techtarget.com/whatis/definition/CamelCase#:~:text=CamelCase%20is%20a%20way%20to,programming%20and%20computer%20naming%20conventions.